### PR TITLE
Add pre-baked Centos Image

### PR DESCRIPTION
**Pull request by kashifest**
This PR adds prebaked Centos node image URL. 
The preBaking is done [here](https://github.com/Nordix/airship-dev-tools/pull/231)

### CODE REVIEW

**Review by Jaakko-Os — COMMENTED** [review]
lgtm

File: vm-setup/roles/v1aX_integration_test/vars/main.yml [hunk 1]
```diff
@@ -60,9 +60,9 @@ IMAGE_LOCATION: "{{ lookup('env', 'IMAGE_LOCATION') | default('https://artifacto
 IMAGE_URL: "http://172.22.0.1/images/{{ RAW_IMAGE_NAME }}"
 IMAGE_CHECKSUM: "http://172.22.0.1/images/{{ RAW_IMAGE_NAME }}.md5sum"
 
-IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_NAME') | default('CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2', true)}}"
+IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_NAME') | default('CENTOS_8.2_NODE_IMAGE_K8S_v1.18.8.qcow2', true)}}"
 RAW_IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_RAW_NAME') | default('CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64-raw.img', true)}}"
```
Comment (fmuyassarov) [inline vm-setup/roles/v1aX_integration_test/vars/main.yml:64]: Should this also be changed to the similar name as the default value of IMAGE_NAME_CENTOS?

Comment (fmuyassarov) [inline vm-setup/roles/v1aX_integration_test/vars/main.yml:64]: Should this also be changed to the similar name`-raw.img `as the default value of IMAGE_NAME_CENTOS?

**Review by fmuyassarov — COMMENTED** [review]
Please make sure to update the vars.md with corresponding changes.Otherwise lgtm.

File: vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc [hunk 2]
```diff
@@ -32,23 +32,6 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
...
-      - >-
-        dnf install gcc kernel-headers kernel-devel keepalived
-        device-mapper-persistent-data lvm2 -y
-      - echo  \"Installing kubernetes binaries\"
-      - curl -L --remote-name-all https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/{kubeadm,kubelet,kubectl}
-      - chmod a+x kubeadm kubelet kubectl
-      - mv kubeadm kubelet kubectl /usr/local/bin/
-      - mkdir -p /etc/systemd/system/kubelet.service.d
-      - retrieve.configuration.files.sh \"https://raw.githubusercontent.com/kubernetes/release/{{ KUBERNETES_BINARIES_CONFIG_VERSION }}/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service\" /etc/systemd/system/kubelet.service
-      - retrieve.configuration.files.sh \"https://raw.githubusercontent.com/kubernetes/release/{{ KUBERNETES_BINARIES_CONFIG_VERSION }}/cmd/kubepkg/templates/latest/deb/kubeadm/10-kubeadm.conf\" /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
```
Comment (maelk) [inline vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc:51]: you need to do something similar for the workers

Comment (kashifest) [reply vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc:51]: Ow, good catch, it might improve the timing more

**Review by namnx228 — APPROVED** [review]

File: vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc (hunk 2 above)
Comment (kashifest) [reply vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc:51]: done

File: vm-setup/roles/v1aX_integration_test/vars/main.yml (hunk 1 above)
Comment (kashifest) [reply vm-setup/roles/v1aX_integration_test/vars/main.yml]: Done

**Review by maelk — APPROVED** [review]

**Comment by kashifest** [comment]
@fmuyassarov Document update done, are those ok or do I need to add something somewhere else also?

**Review by furkatgofurov7 — COMMENTED** [review]
lgtm
Since others have comments, I will leave it to them to lgtm.

**Pull request merged**